src/main/java/com/answerdigital/benhession/academy/answerkingweek2/services/CategoryService.java

```diff
@@ -71,7 +71,7 @@ public Category addItemToCategory(final Long categoryId, final Long itemId) {
             throw new ConflictException(exceptionMessage);
         }
 
-        category.addItem(item);
+        category.getItems().add(item);
```

Comment:
Could we please change this back, I've added new methods as part of BENCH-81 for addItem and removeItem as without it, it didn't work.

Reply:
those methods were not in at the point of writing this, will revert it